### PR TITLE
Add tests for null keyword behavior

### PR DESCRIPTION
**Pull request by adamperlin**
Adds test cases to address #229. Results: `null` currently has no special meaning in flux and is a valid identifier.

### CODE REVIEW

File: query/querytest/test_cases/null_as_ident.flux [hunk 1]
```diff
@@ -0,0 +1,3 @@
+null = "test"
+from(db: null)
```
Comment (aanthony1243) [inline query/querytest/test_cases/null_as_ident.flux]: This doesn't work for a few reasons, first being that a DB is a data source and a null DB doesn't make much sense.   Second, the test framework replaces this line with the CSV data source so it's never executed on.  

after range, add   a   `|> filter(fn: (r) => r._value == null)` 

I would expect this one to error out, actually.

File: query/querytest/test_cases/null_as_value.flux [hunk 2]
```diff
@@ -0,0 +1,3 @@
+db = null
```
Comment (aanthony1243) [inline query/querytest/test_cases/null_as_value.flux]: same thing here, let's put this into a filter function.

**Comment by adamperlin** [comment]
Note: rebased against master

File: query/querytest/test_cases/null_as_ident.flux [hunk 3]
```diff
@@ -0,0 +1,4 @@
+null = 318324641792
```
Comment (aanthony1243) [inline query/querytest/test_cases/null_as_ident.flux]: This is  demonstrating a current behavior, but tests should match desired behavior and not actual behavior.  The second test effectively shows that null is not a reserved keyword.  What should be happening in this case, is an error saying we can't assign a value to null.   I think it's OK to just drop this test.

Comment (adamperlin) [reply query/querytest/test_cases/null_as_ident.flux]: ok, sounds good.
